tests/test_orderbook/conftest.py

```
@@ -0,0 +1,128 @@
+    for definition in await maker_tester_session.client.reference.get_market_definitions():
+        if definition.symbol == symbol:
+            market_def = definition
+            break
+
+    if market_def is None:
+        pytest.skip(f"Perp market {symbol} not present in /v2/marketDefinitions")
+
+    try:
+        oracle_price = float(await maker_tester_session.data.current_price(symbol))
+    except (OSError, RuntimeError, ValueError) as e:
+        logger.warning(f"Failed to fetch oracle price for {symbol}: {e}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)